neurom/apps/morph_stats.py

```diff
@@ -95,11 +95,15 @@ def extract_dataframe(morphs, config, n_workers=1):
     """
     if isinstance(morphs, Morphology):
         morphs = [morphs]
+    elif isinstance(morphs, Population):
+        morphs = morphs._files  # pylint: disable=protected-access
```

Comment:
Wouldn't `morphs.morphologies` do the same?

Reply:
Actually doesn't the `list(map(func, morphs))` below work automatically on the population?
https://github.com/BlueBrain/NeuroM/blob/b2c99d936fabed0fb792e644561e9ccd483d6f5a/neurom/core/population.py#L95

Reply:
As far as I understand it would load the files. The point here is that we need file paths, not morphologies, because `Morphology` objects are not picklable so multiprocessing fails.

Reply:
> Actually doesn't the `list(map(func, morphs))` below work automatically on the population?
> 
> https://github.com/BlueBrain/NeuroM/blob/b2c99d936fabed0fb792e644561e9ccd483d6f5a/neurom/core/population.py#L95

It works properly in the serial case but not in parallel case.

Reply:
Ah I see.

Reply:
Nevertheless, we can move it to the multiproc case only if you prefer

Reply:
No, it's fine like that if it works.

Reply:
Ok!